pkg/resourcecreator/batch/job.go

```diff
@@ -10,8 +10,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func CreateJobSpec(naisjob *nais_io_v1.Naisjob, ast *resource.Ast, resourceOptions resource.Options) (batchv1.JobSpec, error) {
-	podSpec, err := pod.CreateSpec(ast, resourceOptions, naisjob.GetName(), corev1.RestartPolicyNever)
+func CreateJobSpec(naisjob *nais_io_v1.Naisjob, ast *resource.Ast, resourceOptions resource.Options, restartPolicy corev1.RestartPolicy) (batchv1.JobSpec, error) {
+	podSpec, err := pod.CreateSpec(ast, resourceOptions, naisjob.GetName(), restartPolicy)
```

Comment:
I would place the call to RestartPolicy() in here, instead of the outside. Otherwise, LGTM

Reply:
done.